includes/classes/admin/base.php

```diff
@@ -41,6 +41,9 @@ abstract class Base extends Enqueue {
 	 */
 	public $default_options = array();
 
+	/** @var string $logo */
+	protected $logo;
```

Comment:
What this doing as I can't see where it is used?

Reply:
@CWDN @HughbertD 

For this file, this was not an update we made, but from what I'm seeing the "$logo" variable is in use on line 56 of this file

Reply:
@CWDN @pat-kavanagh This looks like it appears on the [main splash screen](https://share.zight.com/rRuDJEdB) - we'll be updating this anyway (I think you might have already done it Chris!)